README.md

```diff
@@ -1,32 +1,30 @@
-Polysh
-~~~~~~
+[![Build Status](https://travis-ci.org/innogames/polysh.svg?branch=master)](https://travis-ci.org/innogames/polysh)
+
+# Polysh
```

Comment:
Should this be a bigger title? or not exists at all?

Reply:
Can you check if you are fine with it now?

Reply:
Yes, I am.  Although this README should have great deal more about using the tool.